File: PlaceYourBets.ConvertedToC#/UserList.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Drawing;
using System.Diagnostics;
using System.Windows.Forms;
using System.Linq;
using System.Xml.Linq;
namespace PlaceYourBets
{
	public class UserList
	{

		public List<Users> stock {
			get { return m_stock; }
			set { m_stock = value; }
		}
		private List<Users> m_stock;
		public int success {
			get { return m_success; }
			set { m_success = value; }
		}

		private int m_success;
	}
}

File: Backup/PlaceYourBets/UserList.vb
﻿Public Class UserList

    Public Property stock() As List(Of Users)
        Get
            Return m_stock
        End Get
        Set(value As List(Of Users))
            m_stock = value
        End Set
    End Property
    Private m_stock As List(Of Users)
    Public Property success() As Integer
        Get
            Return m_success
        End Get
        Set(value As Integer)
            m_success = value
        End Set
    End Property
    Private m_success As Integer

End Class
